src/pipeline_edsl/to_workflow.ml

```diff
@@ -521,6 +537,41 @@ module Make (Config : Compiler_configuration)
     in
     Fastqc_result (Tools.Fastqc.run ~run_with ~fastq ~output_folder)
 
+  let vcf_annotate_polyphen reference_build vcf =
+    let v = get_vcf vcf in
+    let out_filename = Filename.chop_extension v#product#path ^ "_polyphen.vcf" in
+    let output_vcf = Config.work_dir // out_filename in 
```

Comment:
Here `out_filename` is already an absolute path, so `Config.work_dir // out_filename` becomes a wrong path.

Failed with

```
FileNotFoundError: [Errno 2] No such file or directory: '/..../biokepi-work-dir/work/normal-SK-PB22-tumor1-Sl15-noRNA-b37//..../biokepi-work-dir/work/normal-SK-PB22-tumor1-Sl15-noRNA-b37/normal-SK-PB22-_bwamem-default_markdup-0-_indelreal-ignore-mismatch-ignore-mismatch--2x-15c8f576f6fbef757248a5d38cea8e65-merged_bqsr-BdefaultPdefault_gatkhaplo-merged_polyphen.vcf'
```